tools/fast/bin/localnet/README.md

```diff
@@ -0,0 +1,83 @@
+# localnet
```

Comment:
are we linking to this file from anywhere?

Reply:
No, not yet. It may get linked to by a will page at some point I'm sure.